docs/configuration.md

```diff
@@ -2668,6 +2659,21 @@ of the import statement, up to and including the final dot.
 > rule like `OrganizeImports`. However, on a large codebase, the overhead
 > of using semantic `scalafix` rules might be substantial.
 
+This rule will separate all import statements into groups. If sorting is
+enabled (i.e., not `none`), imports will also be sorted within each group.
+
+The rule accepts the following parameters:
+
+- `rewrite.imports.groups`: defines several sets of regular expressions; each
+  set defines a single group, and the groups are output in the order they
+  are configured (imports not matching any of the regexes will form their own
+  group at the end). Regular expressions are applied to the entire parent domain
+  of the import statement, up to and including the final dot.
+- `rewrite.imports.contiguousGroups` (since v3.0.2):
+  - if `no` (default), grouping will happen on all imports within the same
```

Comment:
I would honestly make it a default and not even add a setting for this. There is a high chance that `val` or something else between imports is there on purpose.

Reply:
i can make it default but in our codebase, we prefer to take imports spread out around the file and put them in one place. users can define their vals and then import everything.

Reply:
Ach, ok, it makes sense. But I think the default should be to not touch it, that's the safer side.

Reply:
changed the default

Reply:
Awesome thanks!